src/click/types.py

```diff
@@ -297,16 +297,15 @@ def convert(
         if normed_value in normed_choices:
             return normed_choices[normed_value]
 
+        self.fail(self.get_missing_message(value), param, ctx)
+
+    def get_invalid_choice_fail_message(self, value: t.Any) -> str:
```

Comment:
```suggestion
    def get_invalid_choice_message(self, value: t.Any) -> str:
```

Let's shorten this a bit?